rerun_py/rerun_sdk/rerun/archetypes/transform3d_ext.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from typing import Any
+
+from rerun.datatypes.mat3x3 import Mat3x3Like
+from rerun.datatypes.rotation3d import Rotation3DLike
+from rerun.datatypes.scale3d import Scale3DLike
+from rerun.datatypes.transform3d import Transform3DLike
+from rerun.datatypes.translation_and_mat3x3 import TranslationAndMat3x3
+from rerun.datatypes.translation_rotation_scale3d import TranslationRotationScale3D
+from rerun.datatypes.vec3d import Vec3DLike
+
+
+class Transform3DExt:
+    def __init__(
+        self: Any,
+        transform: Transform3DLike | None = None,
+        *,
+        translation: Vec3DLike | None = None,
+        rotation: Rotation3DLike | None = None,
+        scale: Scale3DLike | None = None,
+        matrix: Mat3x3Like | None = None,
+        from_parent: bool = False,
+    ):
+        """
+        Create a new instance of the Transform3D archetype.
+
+        Parameters
```

Comment:
May be an example section here would be good, as a way to list all supported combinations. This is going to be useful for the user, to avoid solving for constraints listed in the `Parameter` section

Reply:
we generally have examples on the type documentation. I'm hesitant to add examples now and here and only for python.
"all supported combintations" may also be way too much

Reply:
I agree this is not the place for a full on example, but something akin to this would still provide value IMO. It's much more explicit as to what the 3 general forms are.

```
# use one of the concrete transform data type, e.g.
rr.Transform3D(rr.datatypes.TranslationAndMat3x3)

# using translation and/or and 3x3 matrix
rr.Transform3D(translation=[...], mat3x3=[...])

# using any or all of the elementary transform components
rr.Transform3D(translation=[...], rotation=..., scale=...)
```

Reply:
It could even be a bullet list without actual code.